frame/democracy/src/conviction.rs

```diff
@@ -0,0 +1,119 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
full cpy paste

Reply:
It's already full copy paste, do you mean to add something more?

Reply:
may be some how indicate in somewhere in repo how clone wasa produced and what was changed

Reply:
sure, will take care with the documentation